geps/gep-1619.md

```diff
@@ -42,6 +42,8 @@ Session persistence can introduce security and privacy vulnerabilities if not pr
 
 To mitigate these security concerns, it is important to implement session persistence using secure practices, such as using strong session ID generation algorithms, implementing session timeouts, encrypting sensitive data, and monitoring server resources for unusual activity.
 
+IP address reuse may also be a security or privacy concern when using session persistence or session affinity. If Kubernetes reuses an IP address of previously shutdown pod, the new pod may receive session persistent traffic meant for the old pod.
```

Comment:
@sunjayBhatia regarding https://github.com/kubernetes-sigs/gateway-api/pull/1935#issuecomment-1523022313

Reply:
Got it.

You probably tagged the wrong person - assuming you were trying to tag me (sanjaypujare vs sunjayBhatia)